docs/plugins/mailchimp.rst

```diff
@@ -3,4 +3,47 @@
 MailChimp
 #########
 
-.. vale on
\ No newline at end of file
+.. vale on
+This plugin can send contacts to MailChimp cilents based on Contact actions or Point Triggers.
+
+## Authorize
+
+### Get MailChimp API key
+
+1. Create a MailChimp account.
+2. Go to *Account* / *Extras* / *API Keys* and create a new one.
+3. Copy the created API Key.
+
+### Authorize Mautic - MailChimp plugin
+
+1.Fill in with your MailChimp's account **username** 
+2. Add the **API key**
+3. Click on ***Save & Close***  
+
+## Configure the plugin
+
+Navigate to the *Features* tab in the plugin configuration modal box. You should see this note:
+
+> The Contact Field Mapping tab will appear after selecting a segment and will update after changing the selected segment.
+
+![MailChimp Plugin configuration](plugins-mailchimp-configure.png "MailChimp Plugin configuration")
```

Comment:
This bit also needs converting to the RST way of doing images: 

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/images.html